dms-service/README.md

```diff
@@ -62,6 +62,7 @@ swim:
       filename-to-coo: # for coo-source filename_map
       filename-overwrite-pattern: # overwrite ContentObject name via Regex pattern
       incoming-name-pattern: # overwrite Incoming name via Regex pattern, only applies to type incoming_object
+      metadata-subject: # enables incoming subject be built from metadata file
```

Comment:
In addition to this line, there should also be something about the new functionality in section `Metadata file` further down in the README - the fields should be visible in the Metadata-JSON there and some description should exist.